src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -0,0 +1,335 @@
+        legend({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap', marginTop: '0.25rem', float: 'left' } },
+          [span({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap' } },
+            ['Add ', span({ style: { fontWeight: 'bold' } }, 'terra-billing@terra.bio'), ' as a Billing Account User',
+              span({ style: { fontWeight: 'bold' } }, ' to your billing account.')]),
+          div({ style: { marginTop: '3rem' } }, [linkToSupport])]),
+        div({ style: { width: '25%', float: 'right' } }, [
+          div({ style: { display: 'flex', displayDirection: 'row' } }, [
+            h(RadioButton, {
+              text: "I don't have access to do this", name: 'permission',
+              checked: accessToAddBillingAccountUser === false,
+              disabled: !isDone && !isActive,
+              labelStyle: { ...styles.radioButtonLabel },
+              onChange: () => {
+                if (isDone) {
+                  setActiveStep(3)
+                  setRefreshed(false)
+                }
+                setAccessToAddBillingAccountUser(false)
+              }
+            })
+          ]),
+          div({ style: { marginTop: '2rem', display: 'flex', displayDirection: 'row' } }, [
+            h(RadioButton, {
+              text: 'I have added terra-billing as a billing account user (requires reauthentication)', name: 'permission',
+              checked: accessToAddBillingAccountUser === true,
+              disabled: !isDone && !isActive,
+              labelStyle: { ...styles.radioButtonLabel },
+              onChange: async () => {
+                if (isDone) {
+                  setRefreshed(false)
+                } else {
+                  await authorizeAndLoadAccounts()
+                  next()
+                }
+                setAccessToAddBillingAccountUser(true)
+              }
+            })
+          ])
+        ])
+      ])
+
+    const dontHaveAccessToAdd =
+      fieldset({ style: { border: 'none', margin: 0, padding: 0, display: 'block' } }, [
+        legend({ style: { width: '55%', fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap', marginTop: '0.25rem', float: 'left' } },
+          [
+            span(['Contact your billing account administrator and have them add you and ',
+              span({ style: { fontWeight: 'bold' } }, ['terra-billing@terra.bio']), ' as a Billing Account User',
+              span({ style: { fontWeight: 'bold' } }, [' to your organization\'s billing account.'])]),
+            div({ style: { marginTop: '2rem' } }, [linkToSupport])
+          ]),
+        checkbox
+      ])
+
+    return li({ 'aria-current': isActive ? 'step' : false, style: { ...styles.stepBanner(isActive), display: 'flex', flexDirection: 'column' } }, [
+      h3({ style: { fontSize: 18, marginTop: 0 } }, ['STEP 3']),
+      isActive || isDone ?
+        (accessToBillingAccount ?
+          (accessToAddBillingAccountUser !== undefined ?
```

Comment:
The other logic is correct except for the first one: 
> if this is the active step (isActive), or we're beyond this step (isDone) -> we render haveABillingAccount

If this is not the active step or if it is not done -> we render haveABillingAccount

Hmm, I don't think this works because the first logic is incorrect.
`isActive || isDone || (accessToBillingAccount && accessToAddBillingAccountUser) ? haveABillingAccount : dontHaveAccessToAdd`

Reply:
Sorry, reading it again, I think the other logic might also be incorrect. This is what should happen: 

- if we're on steps 1 or two (both `isActive` and `isDone` are false) -> we render 
`haveABillingAccount`   
- if we're on step 3 or 4 (if `isActive` is true or `isDone` is true), and accessToBillingAccount is true and accessToAddBillingAccountUser is undefined (not selected yet)-> we render 
`haveABillingAccount` 
- if we're on step 3 or 4 (if `isActive` is true or `isDone` is true), and accessToBillingAccount is true and accessToAddBillingAccountUser is true-> we still render 
`haveABillingAccount` 
- if we're on step 3 or 4 (if `isActive` is true or `isDone` is true), and accessToBillingAccount is true and accessToAddBillingAccountUser is false (selected)-> we render 
`dontHaveAccessToAdd` 
- if we're on step 3 or 4 (if `isActive` is true or `isDone` is true), and accessToBillingAccount is false and accessToAddBillingAccountUser is undefined-> we render
`dontHaveAccessToAdd` 

I simplified the logic to this, but wasn't sure if this makes it easier to understand. Is there a way to further simplify/clarify this: 
```
   (isActive || isDone) &&
      (!accessToBillingAccount || (accessToAddBillingAccountUser !== undefined && !accessToAddBillingAccountUser)) ?
        dontHaveAccessToAdd : haveABillingAccount
```